controllers/export-data.js

```diff
@@ -449,38 +449,51 @@ var outputToCsv = function(options, tabs, callback) {
 };
 
 var outputToXml = function(options, tabs, callback) {
+  callback(null, function(write, done) {
+    var workbook = xmlbuilder.begin({ allowSurrogateChars: true, allowEmpty: true }, write)
+      .dec({ encoding: 'UTF-8' })
+      .ele('Workbook')
+      .att('xmlns', 'urn:schemas-microsoft-com:office:spreadsheet')
+      .att('xmlns:o', 'urn:schemas-microsoft-com:office:office')
+      .att('xmlns:x', 'urn:schemas-microsoft-com:office:excel')
+      .att('xmlns:html', 'http://www.w3.org/TR/REC-html140')
+      .att('xmlns:ss','urn:schemas-microsoft-com:office:spreadsheet')
+      .ins('mso-application', 'progid="Excel.Sheet"');
+
+    var row;
+
+    tabs.forEach(function(tab) {
+      var table = workbook
+        .ele('Worksheet', { 'ss:Name': tab.name })
+        .ele('Table');
+
+      if (!options.skipHeader) {
+        row = table.ele('Row');
+        tab.columns.forEach(function(column) {
+          row.ele('Cell').ele('Data', {'ss:Type': 'String'}, column.label);
+          row.up().up();
```

Comment:
Main additions here is the use of `up()` to move the 'pointer' back to the correct place. This is required for streaming as opposed to keeping the whole thing in memory, otherwise invalid XML is generated.